flags_test.go

```diff
@@ -387,6 +394,47 @@ func TestHandleDeactivateCommandWithForce(t *testing.T) {
 	assert.Equal(t, expected, flags.Command)
 }
 
+func TestHandleLocalDeactivationWithControlModeNotOne(t *testing.T) {
```

Comment:
Instead of ControlModeNotOne, this doesnt tell me really what we're testing. What does NotOne mean?